source/Nevermore/QueryBuilderWhereExtensions.cs

```diff
@@ -1,40 +1,58 @@
 using System;
+using System.Collections;
+using System.Collections.Generic;
 using System.Linq;
 using System.Linq.Expressions;
 using System.Reflection;
+using Nevermore.AST;
 
 namespace Nevermore
 {
+    public enum SqlOperand
```

Comment:
Should this be removed as it has been superseded? It should be easy to find-replace instances of this.

Reply:
Yes I only left it for backwards compatibility. Happy to remove it though.

Reply:
Probably for the better to reduce confusion given the ease and reliability of conversion

Reply:
:+1: The only problem is `Contains`, `StartsWith` and `EndsWith` which was some sugar that we added on top of sql (change the way we formatted the parameter values, rather than changing the actual sql query). I should be able to migrate this to using the new linq expression extension methods though.

Reply:
Done